projects/aca-content/src/lib/components/sidenav/sidenav.component.spec.ts

```diff
@@ -44,13 +46,18 @@ describe('SidenavComponent', () => {
             appNavNarMode$: new BehaviorSubject('expanded'),
             toggleAppNavBar$: new Subject()
           }
+        },
+        {
+          provide: SidenavLayoutComponent,
```

Comment:
Because provide and useClass are the same - can not you just put only class name in providers? Like: 

providers: [SidenavLayoutComponent]

Reply:
Updated, ty